src/averageAges.js

```diff
@@ -15,11 +16,15 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = century
+    ? people.filter((p) => p.sex === 'm' && Math.ceil(p.died / 100) === century)
+    : people.filter((p) => p.sex === 'm');
+
+  let sum = 0;
+
+  men.forEach((m) => sum += m.died - m.born);
```

Comment:
Use `reduce` method instead. It's faster and exactly fit for this situation. 
Also it's more readable to use destructuring wherever it possible below and above. Something like this:
`
const totalAge = men.reduce((sum, { born, died }) => sum + (died - born), 0);
`

Please, find all same cases in this code and figure it out.

And please, take into account declaration names, 'p' and 'm' isn't good naming, it takes much more attention than it needs